delegatable_credentials/src/msbm/issuance.rs

```diff
@@ -18,7 +18,7 @@ use ark_std::{rand::RngCore, vec::Vec, UniformRand};
 /// of attributes
 #[derive(Clone, Debug)]
 pub struct Credential<E: Pairing> {
-    pub max_attributes_per_commitment: usize,
+    pub max_attributes_per_commitment: u64,
```

Comment:
`u16` is a good enough upper bound on the number of attributes. Also update type of `max_attributes_per_commitment` in `issue_*`